.gitignore

```diff
@@ -301,3 +301,6 @@ $RECYCLE.BIN/
 
 # Project Files
 modules/telestion-mavlink/src/main/messages/org/telestion/mavlink/
+
+# Python things
```

Comment:
This change should better go into its own PR.

Reply:
Will do in the future :D